### ISSUE TITLE
Importing with new Unique IDs resulted in 3 pending resources

### ISSUE THREAD

**Issue by clapierre** [benetech/ImageShare#177]
@jkva when I deleted all resource and resource files from Imageshare Dev environment, and deleted them all from Trash when I re-imported the 1193 DCMP videos all but 3 resources were imported with status draft.  3 of them were marked as Pending.  I have no idea why, maybe they were pending as test previously but when I deleted all the resources and removed them from trash I wouldn't expect any of that state to remain.

Here is a screen shot of the 3 which were put as pending, which I am pretty sure I was playing with those video status's a few weeks ago.

![Screen Shot 2020-07-10 at 12 11 27 PM](https://user-images.githubusercontent.com/7736749/87203004-8e922400-c2b6-11ea-95fc-b085b90e6e13.png)

**Comment by clapierre** [comment]
@jkva Ok I figured out how this happened and I think its the correct behavior.
I imported first ImageShareMaster.json which had those 3 resources in there as test files for DCMP.  I then published those resources.  I then when in and imported all 1193 DCMP videos which included those 3 DCMP, so the status went from Published to Pending.  I think is the correct behavior, and if you agree we can close this issue.
